### PR TITLE
Niche targeting for marketers

### PR DESCRIPTION
**Pull request by davidfischer**
Add a marketer focused post and crosslink to the dev post.

### CODE REVIEW

File: content/posts/2024-niche-ad-targeting.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ and then storing the topics in our database for fast ad serving.
 This gave us a good starting point for targeting ads by topic,
 but we wanted to get more granular.
 
+*Just a quick aside. This post is aimed at developers.
+If you're interested in one aimed at marketers, check out our
+[developer-focused post]({filename}../posts/2024-niche-ad-targeting.md).*
```
Comment (ericholscher) [inline content/posts/2024-niche-ad-targeting.md:22]:
```suggestion
*This post is written for developers, but if you're interested in how this applies toΩ marketers, check out our
[marketing-focused post]({filename}../posts/2024-niche-ad-targeting.md).*
```

File: content/posts/2024-niche-targeting-marketing-post.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+Title: AI-Powered Niche Targeting for Developer Marketers
```
Comment (ericholscher) [inline content/posts/2024-niche-targeting-marketing-post.md:1]: The filename here feels weird? We should do something like `niche-targeting-for-marketers`.

File: content/posts/2024-niche-targeting-marketing-post.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+without needing to know anything about the user themselves.
+So yes, AI-targeted ads can indeed be great for privacy.
+It's like having your cake and eating it too, but without anyone spying on your cake preferences.
+
+
+## Niche targeting prototype and case study
+
+While we're still building and refining the capabilities for this targeting,
+we wanted to show some early examples of just how it will work.
+Here's an [example](https://www.ethicalads.io/advertisers/similar-pages/?url=https%3A%2F%2Fwww.mongodb.com%2Fatlas) where you can see how we're able to target ads
```
Comment (ericholscher) [inline content/posts/2024-niche-targeting-marketing-post.md:56]:
```suggestion
Here's a [niche targeting example](https://www.ethicalads.io/advertisers/similar-pages/?url=https%3A%2F%2Fwww.mongodb.com%2Fatlas) where you can see how we're able to target ads
```

**Review by ericholscher — APPROVED** [review]
This seems great 💯

File: content/posts/2024-niche-targeting-marketing-post.md (hunk 2 above)
Comment (davidfischer) [reply content/posts/2024-niche-targeting-marketing-post.md]: Renamed

**Comment by davidfischer** [comment]
Issues are fixed up and I'm aiming for a Tuesday merge.

**Pull request merged**